markdownlint-cli2-config-schema.json

```diff
@@ -0,0 +1,83 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"title": "markdownlint-cli2 configuration schema",
+	"type": "object",
+	"properties": {
+		"config": {
+			"$ref": "./node_modules/markdownlint/schema/markdownlint-config-schema.json"
```

Comment:
I love this. I wish I could do the same in package.json, but it doesn't seem like I can. If you see how, it'd be nice to not duplicate the schema there.

Reply:
Will this be correct after bundling? Seems like it will for the same reason the existing reference works. But it'd be nice to have you run bundled and confirm.

Reply:
I've only been testing my changes by bundling and launching the extension and manipulating Markdown files and config files. Here's a screenshot of the `$ref` working within a `.markdownlint-cli2.jsonc` file:

<img width="764" alt="Screen Shot 2020-11-28 at 7 05 04 PM" src="https://user-images.githubusercontent.com/831617/100530261-0bae2f00-31ad-11eb-875a-8abd2c626c7b.png">

I was going to make an attempt at using `$ref` inside of `package.json` later. What issues did you run into?

Reply:
User error? :)